meshtastic/__init__.py

```diff
@@ -462,16 +462,20 @@ def _handlePacketFromRadio(self, meshPacket):
             # byte array.
             asDict["decoded"]["data"]["payload"] = meshPacket.decoded.data.payload
 
-            # UNKNOWN_APP is the default protobuf pornum value, and therefore if not set it will not be populated at all
+            # UNKNOWN_APP is the default protobuf portnum value, and therefore if not set it will not be populated at all
             # to make API usage easier, set it to prevent confusion
             if not "portnum" in asDict["decoded"]["data"]:
-                asDict["decoded"]["data"]["portnum"] = portnums_pb2.PortNum.UNKNOWN_APP
+                asDict["decoded"]["data"]["portnum"] = portnums_pb2.PortNum.Name(portnums_pb2.PortNum.UNKNOWN_APP)
 
             portnum = asDict["decoded"]["data"]["portnum"]
+
+            if isinstance(portnum, str):
+                portnum = portnums_pb2.PortNum.Value(portnum)
```

Comment:
@timgunter what do you think about we take this PR except for these two lines?  Then (it would seem to me - I haven't yet tried this) we would get a string identifier for well-known portnums (for nice readability/logging).  For all other cases the identifier would come through as a number and you'd subscribe for "meshtastic.receive.data.4403" etc...

I think that it is a bit odd to have the IDs come back as strings 'sometimes' but I've kinda been digging it and they are just IDs.

Thoughts?